packages/ui/i18n/en.pot

```diff
@@ -5,8 +5,17 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2025-10-31T13:42:38.500Z\n"
-"PO-Revision-Date: 2025-10-31T13:42:38.501Z\n"
+"POT-Creation-Date: 2025-12-09T10:09:29.248Z\n"
+"PO-Revision-Date: 2025-12-09T10:09:29.249Z\n"
+
+msgid "Median prediction"
+msgstr "Median prediction"
+
+msgid "80% prediction interval"
```

Comment:
@eirikhaugstulen I'm just curious about the QUANTILE_MAP in https://github.com/dhis2-chap/chap-frontend/blob/main/packages/ui/src/utils/PredictionViewModel.ts#L9

These quantiles are totally unrelated to the Prediction interval, right?